### ISSUE TITLE
Cannot load PGN using starting position

### ISSUE THREAD

**Issue by dlbbld** [bhlangonijr/chesslib#53]
When loading a PGN which uses a starting position, there is an exception, e.g.

```
[Event "?"]
[Site "?"]
[Date "????.??.??"]
[Round "?"]
[White "?"]
[Black "?"]
[Result "*"]
[SetUp "1"]
[FEN "3k4/8/8/8/3K4/3R4/8/8 w - - 15 100"]

100. Re3 Kc7 101. Re5 Kc6 *
```

**Comment by bhlangonijr** [comment]
Fixed in 1.2.4

**Comment by dlbbld** [comment]
I checked a few examples and they worked as expected.
